[2023-06-11T00:10:14.692+0200] {processor.py:156} INFO - Started process (PID=4376) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:10:14.696+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:10:14.700+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:10:14.699+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:10:14.742+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:10:14.734+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:10:14.745+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:10:14.982+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.329 seconds
[2023-06-11T00:12:46.422+0200] {processor.py:156} INFO - Started process (PID=4633) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:12:46.438+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:12:46.441+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:12:46.440+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:12:46.541+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:12:46.491+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:12:46.547+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:12:46.763+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.375 seconds
[2023-06-11T00:14:04.756+0200] {processor.py:156} INFO - Started process (PID=4669) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:14:04.761+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:14:04.763+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:14:04.762+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:14:04.789+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:14:04.771+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:14:04.790+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:14:04.932+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.183 seconds
[2023-06-11T00:15:14.401+0200] {processor.py:156} INFO - Started process (PID=4709) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:15:14.407+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:15:14.409+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:15:14.409+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:15:14.443+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:15:14.419+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:15:14.446+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:15:14.607+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.222 seconds
[2023-06-11T00:16:38.209+0200] {processor.py:156} INFO - Started process (PID=4756) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:16:38.214+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:16:38.216+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:16:38.215+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:16:38.231+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:16:38.222+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:16:38.232+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:16:38.280+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.078 seconds
[2023-06-11T00:17:27.502+0200] {processor.py:156} INFO - Started process (PID=4793) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:17:27.513+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:17:27.516+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:17:27.516+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:17:27.557+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:17:27.552+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:17:27.558+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:17:27.690+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.233 seconds
[2023-06-11T00:18:16.397+0200] {processor.py:156} INFO - Started process (PID=4829) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:16.402+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:18:16.403+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:18:16.403+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:16.409+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:18:16.407+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:18:16.410+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:16.483+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.093 seconds
[2023-06-11T00:18:58.230+0200] {processor.py:156} INFO - Started process (PID=4862) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:58.237+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:18:58.238+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:18:58.238+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:58.246+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:18:58.243+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:18:58.247+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:18:58.293+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.069 seconds
[2023-06-11T00:20:27.768+0200] {processor.py:156} INFO - Started process (PID=4908) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:20:27.781+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:20:27.783+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:20:27.782+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:20:27.802+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:20:27.787+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:20:27.803+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:20:27.893+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.132 seconds
[2023-06-11T00:21:26.964+0200] {processor.py:156} INFO - Started process (PID=4943) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:21:26.972+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:21:26.974+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:21:26.973+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:21:26.984+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:21:26.981+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:21:26.985+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:21:27.047+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.091 seconds
[2023-06-11T00:22:35.478+0200] {processor.py:156} INFO - Started process (PID=4979) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:22:35.484+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:22:35.486+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:22:35.486+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:22:35.496+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:22:35.493+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:22:35.496+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:22:35.560+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.096 seconds
[2023-06-11T00:23:31.260+0200] {processor.py:156} INFO - Started process (PID=5013) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:23:31.270+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:23:31.271+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:23:31.271+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:23:31.281+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:23:31.276+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:23:31.282+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:23:31.601+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.349 seconds
[2023-06-11T00:24:48.204+0200] {processor.py:156} INFO - Started process (PID=5050) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:24:48.209+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:24:48.220+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:24:48.220+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:24:48.435+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:24:48.237+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:24:48.436+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:24:48.876+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.704 seconds
[2023-06-11T00:26:29.440+0200] {processor.py:156} INFO - Started process (PID=5085) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:26:29.448+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:26:29.450+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:26:29.449+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:26:29.463+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:26:29.460+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:26:29.464+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:26:29.543+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.117 seconds
[2023-06-11T00:27:27.137+0200] {processor.py:156} INFO - Started process (PID=5122) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:27:27.144+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:27:27.146+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:27:27.145+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:27:27.155+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:27:27.153+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:27:27.156+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:27:27.210+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.080 seconds
[2023-06-11T00:28:27.922+0200] {processor.py:156} INFO - Started process (PID=5158) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:28:27.927+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:28:27.930+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:28:27.929+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:28:27.945+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:28:27.943+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:28:27.946+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:28:27.996+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.085 seconds
[2023-06-11T00:29:23.782+0200] {processor.py:156} INFO - Started process (PID=5192) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:29:23.787+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:29:23.788+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:29:23.788+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:29:23.799+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:29:23.793+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:29:23.800+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:29:23.868+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.094 seconds
[2023-06-11T00:30:34.608+0200] {processor.py:156} INFO - Started process (PID=5229) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:30:34.614+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:30:34.616+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:30:34.615+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:30:34.622+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:30:34.620+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:30:34.623+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:30:34.696+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.095 seconds
[2023-06-11T00:31:33.704+0200] {processor.py:156} INFO - Started process (PID=5264) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:31:33.711+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:31:33.713+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:31:33.713+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:31:33.720+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:31:33.717+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:31:33.721+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:31:33.785+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.088 seconds
[2023-06-11T00:32:29.158+0200] {processor.py:156} INFO - Started process (PID=5299) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:32:29.163+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:32:29.165+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:32:29.165+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:32:29.176+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:32:29.171+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:32:29.177+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:32:29.233+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.087 seconds
[2023-06-11T00:33:22.132+0200] {processor.py:156} INFO - Started process (PID=5335) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:33:22.135+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:33:22.137+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:33:22.137+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:33:22.145+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:33:22.142+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:33:22.146+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:33:22.205+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.081 seconds
[2023-06-11T00:34:20.386+0200] {processor.py:156} INFO - Started process (PID=5368) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:34:20.401+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:34:20.404+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:34:20.404+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:34:20.452+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:34:20.436+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:34:20.454+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:34:20.768+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.427 seconds
[2023-06-11T00:35:25.612+0200] {processor.py:156} INFO - Started process (PID=5409) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:35:25.618+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:35:25.619+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:35:25.619+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:35:25.625+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:35:25.624+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:35:25.626+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:35:25.670+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.064 seconds
[2023-06-11T00:36:09.345+0200] {processor.py:156} INFO - Started process (PID=5442) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:09.353+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:36:09.354+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:36:09.354+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:09.361+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:36:09.359+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:36:09.362+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:09.405+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.067 seconds
[2023-06-11T00:36:53.899+0200] {processor.py:156} INFO - Started process (PID=5476) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:53.918+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:36:53.919+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:36:53.919+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:53.931+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:36:53.929+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:36:53.932+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:36:54.329+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.447 seconds
[2023-06-11T00:37:40.122+0200] {processor.py:156} INFO - Started process (PID=5510) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:37:40.130+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:37:40.135+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:37:40.134+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:37:40.157+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:37:40.149+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:37:40.159+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:37:40.231+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.128 seconds
[2023-06-11T00:38:31.217+0200] {processor.py:156} INFO - Started process (PID=5544) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:38:31.230+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:38:31.233+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:38:31.232+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:38:31.256+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:38:31.250+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:38:31.259+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:38:31.337+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.141 seconds
[2023-06-11T00:39:15.441+0200] {processor.py:156} INFO - Started process (PID=5579) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:15.448+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:39:15.449+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:39:15.449+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:15.474+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:39:15.461+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:39:15.475+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:15.558+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.131 seconds
[2023-06-11T00:39:58.437+0200] {processor.py:156} INFO - Started process (PID=5613) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:58.447+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:39:58.449+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:39:58.449+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:58.473+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:39:58.470+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:39:58.473+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:39:58.587+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.164 seconds
[2023-06-11T00:40:41.485+0200] {processor.py:156} INFO - Started process (PID=5647) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:40:41.491+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:40:41.493+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:40:41.492+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:40:41.500+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:40:41.498+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:40:41.501+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:40:41.548+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.070 seconds
[2023-06-11T00:41:24.415+0200] {processor.py:156} INFO - Started process (PID=5681) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:41:24.421+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:41:24.423+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:41:24.422+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:41:24.429+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:41:24.427+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:41:24.431+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:41:24.483+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.076 seconds
[2023-06-11T00:42:11.537+0200] {processor.py:156} INFO - Started process (PID=5719) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:42:11.548+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:42:11.550+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:42:11.550+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:42:11.563+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:42:11.557+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:42:11.565+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:42:11.830+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.300 seconds
[2023-06-11T00:43:00.846+0200] {processor.py:156} INFO - Started process (PID=5753) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:00.858+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:43:00.860+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:43:00.859+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:00.868+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:43:00.866+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:43:00.872+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:00.949+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.119 seconds
[2023-06-11T00:43:44.155+0200] {processor.py:156} INFO - Started process (PID=5787) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:44.159+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:43:44.160+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:43:44.160+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:44.167+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:43:44.164+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:43:44.167+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:43:44.222+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.074 seconds
[2023-06-11T00:44:32.831+0200] {processor.py:156} INFO - Started process (PID=5821) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:44:32.857+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:44:32.866+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:44:32.865+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:44:32.932+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:44:32.906+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:44:32.935+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:44:33.117+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.633 seconds
[2023-06-11T00:45:21.824+0200] {processor.py:156} INFO - Started process (PID=5856) to work on /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:45:21.853+0200] {processor.py:758} INFO - Processing file /home/nanah/airflow/dags/my_first_dag.py for tasks to queue
[2023-06-11T00:45:21.869+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:45:21.857+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:45:21.918+0200] {logging_mixin.py:117} INFO - [2023-06-11T00:45:21.902+0200] {dagbag.py:330} ERROR - Failed to import: /home/nanah/airflow/dags/my_first_dag.py
Traceback (most recent call last):
  File "/home/nanah/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/nanah/airflow/dags/my_first_dag.py", line 1, in <module>
    from dags.lib.index_elasticsearch import index_elasticsearch
ModuleNotFoundError: No module named 'dags'
[2023-06-11T00:45:21.920+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/nanah/airflow/dags/my_first_dag.py
[2023-06-11T00:45:22.222+0200] {processor.py:178} INFO - Processing /home/nanah/airflow/dags/my_first_dag.py took 0.449 seconds
